scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -750,12 +752,37 @@ cleanup_stobs_dir() {
     [[ $REMOTE_STORAGE_STATUS -eq 0 ]] || run_cmd_on_local_node "umount $FAILOVER_MD_DIR" > /dev/null
 }
 
+#The following command gives us the file count on the local node.
+#It takes 2 parameters as input
+#1. filetype : The type of file whose quantity we want to count. Example "m0trace"
+#2. directory : The location where the files of filetype are stored. Example "/var/motr/datarecovery"
+get_file_count() {
+    local file_type=$1
+    local directory=$2
+    local start_time=$3
+    cmd=$(cd $directory; find . -type f -exec stat  -c "%n %Y" {} \;| sort -n | grep $file_type | awk '{if($2>"'$start_time'") print $2; }' | wc -l)
```

Comment:
please rename "cmd" to "count", or "filecount"

Reply:
Acknowledged with commit 6a6078c.